apis/core/lib/domain/table/create.ts

```diff
@@ -30,13 +31,41 @@ export async function createTable({
   const table = await store
     .createMember(tableCollection.term, id.table(csvMapping.term, label.term.value))
 
+  const csvSourceId = resource.out(cc.csvSource).term! as NamedNode
+  const csvSource = await store.get(csvSourceId)
+
   table.addOut(rdf.type, cc.Table)
-  table.addOut(cc.csvSource, resource.out(cc.csvSource))
+  table.addOut(cc.csvSource, csvSourceId)
   table.addOut(cc.csvMapping, csvMapping.term)
   table.addOut(schema.name, label)
   table.addOut(cc.identifierTemplate, resource.out(cc.identifierTemplate))
   table.addOut(schema.color, resource.out(schema.color))
 
+  // Create default column mappings for provided columns
+  resource.out(csvw.column).terms
+    .forEach((columnId) => {
+      const column = csvSource.out(csvw.column).toArray()
+        .find(({ term }) => term.equals(columnId))
+
+      if (!column) {
+        throw new Error(`Column ${columnId} not found`)
+      }
+
+      const columnName = column.out(schema.name).value!
+
+      table.addOut(cc.columnMapping, id.columnMapping(table, columnName), (columnMapping) => {
+        columnMapping
+          .addOut(rdf.type, cc.ColumnMapping)
+          .addOut(cc.sourceColumn, column)
+          .addOut(cc.targetProperty, defaultProperty(columnName))
```

Comment:
I propose the column mappings to be separate resources

Reply:
Can you give more context as to what that would imply?

As I understand, that's required if we want to be able to create/update/delete column mappings through the API without going through the table? But I do I make it happen?

Reply:
`store.create` returns a pointer to a new graph, an independent resource. Create it first; add values; attach only the ID to the table

```suggestion
      const columnMapping = store.create(id.columnMapping(table, columnName))
      columnMapping
        .addOut(rdf.type, cc.ColumnMapping)
        .addOut(cc.sourceColumn, column)
        .addOut(cc.targetProperty, defaultProperty(columnName))

      table.addOut(cc.columnMapping, columnMapping)
```

Reply:
I extracted this in a separate function, as I feel like we're going to need it pretty soon. But now it's not adding the properties on the column mapping and I need to figure out why.